tests/ParserTest.php

```diff
@@ -2,122 +2,220 @@
 namespace DemandwareXml\Test;
 
 use DemandwareXml\Parser;
+use DemandwareXml\Parser\{
+    AssignmentNodeParser,
+    BundleNodeParser,
+    BundleSimpleNodeParser,
+    CategoryNodeParser,
+    CategorySimpleNodeParser,
+    ProductNodeParser,
+    ProductSimpleNodeParser,
+    SetNodeParser,
+    SetSimpleNodeParser,
+    VariationNodeParser,
+    VariationSimpleNodeParser
+};
 use PHPUnit\Framework\TestCase;
+use stdClass;
 
 // note: don't need to parse variants, so no test for those!
 // rebuild fixtures: file_put_contents(__DIR__ . '/fixtures/categories.json', json_encode($parser->categories(), JSON_PRETTY_PRINT) . PHP_EOL);
 class ParserTest extends TestCase
 {
     use FixtureHelper;
 
-    public function testMixedParser()
+    /**
+     * @expectedException              \DemandwareXml\XmlException
+     * @expectedExceptionMessageRegExp /Fatal: xmlParseEntityRef: no name in invalid-products.xml/
```

Comment:
Is this the entire error message? If so, we can do `@expectedExceptionMessage Fatal: xmlParseEntityRef: no name in invalid-products.xml`

Reply:
No, it's not the entire message. There's some info after that seems to differ slighty by platform, hence the regexp.